semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -80,7 +80,6 @@ class MarkdownRenderer:
     ]
 
     prelude = "<!-- semconv {} -->\n"
-    table_headers = "| Attribute  | Type | Description  | Examples  | Requirement Level |\n|---|---|---|---|---|\n"
     table_headers_omitting_req_level = (
         "| Attribute  | Type | Description  | Examples  |\n|---|---|---|---|\n"
```

Comment:
"Type" could also be linked to the list of attribute types.

Reply:
Added the link. We don't have a dedicated section that described attribute type, so added link to https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/README.md#attribute. Totally open to revert this change.

Reply:
I think that is good enough for me!